.github/workflows/aptos-light-client-tag-comparison.yml

```
@@ -7,36 +7,36 @@ on:
 jobs:
   check-tags:
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # Fetch all history so we can perform a rebase
-          ref: "dev"
-
-      - name: Check branch name
-        run: |
-          if [[ "${GITHUB_REF#refs/heads/}" != "dev" ]]; then
-            echo "Workflow run cancelled because it's not on 'dev' branch."
-            exit 1
-          fi
 
       - name: Fetch latest tag
         id: latest-tag
         run: |
           LATEST_TAG=$(curl --silent "https://api.github.com/repos/aptos-labs/aptos-core/releases" | jq -r '.[] | select(.name | ascii_downcase | contains("mainnet")) | .tag_name' | grep '^aptos-node-v' | sort -V | tail -n 1)
-          CURRENT_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
-          if [[ $(echo -e "$CURRENT_TAG\n$LATEST_TAG" | sort -V | tail -n 1) != $CURRENT_TAG ]]; then
-            echo "name=tag::${LATEST_TAG}" | tee -a $GITHUB_OUTPUT
-          else
+          CURRENT_TAG=$(curl --silent "https://api.github.com/repos/samuelburnham/aptos-core/releases" | jq -r '.[] | .tag_name' | grep '^aptos-node-v' | sort -V | tail -n 1)
```

Should this be lurk-lab repo? Can we use `github.repository?
```suggestion
          CURRENT_TAG=$(curl --silent "https://api.github.com/repos/${{ github.repository }}/releases" | jq -r '.[] | .tag_name' | grep '^aptos-node-v' | sort -V | tail -n 1)
```